src/pages/Main/BooksContainer/BooksContainer.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import BookItem from './BookItem/BookItem';
+import styled from 'styled-components';
+
+const BooksContainer = () => {
+  return (
+    <Background>
+      <BookList>
+        <BookItem />
+        <BookItem />
+        <BookItem />
+        <BookItem />
+        <BookItem />
+        <BookItem />
+        <BookItem />
+        <BookItem />
+        <BookItem />
+        <BookItem />
+        <BookItem />
+        <BookItem />
```

Comment:
어차피 구현중인 부분이기 떄문에 임시로 하드코딩으로 넣어서 확인할 수 있긴 하지만..
임시로 작성한 부분이더라도 기왕이면 map 메서드 써서 작성해보시는 건 어떨까요..? Array 생성자 함수와 map 메서드 사용해볼 수 있을 것 같습니다.
참고만 해주세요!
```suggestion
        {[...Array(12)].map((_, idx) => (
          <BookItem key={idx} />
        ))}
```